_episodes/06-agile-in-research.md

```diff
@@ -52,6 +52,10 @@ and it makes more sense to assign particular tasks to members with a relevant ex
 The goal of the Agile way of working is to maximize value for the customer,
 but what is "value" in research? And who is the customer?
 It is often hard/awkward to define scientific outcomes as products with a certain functionality.
+### Open Source software development challenge
+The nature of open source projects is that they draw (and benefit from) collaborators outside of your team, workplace and even country.
+Starting and contributing to open source software projects therefore leads to many dependencies on collaborators outside of the team.
+Not completely "owning" your product is problematic in many Agile methodologies. Approaches like Scrum may not work well or at all in such a situation.
```

Comment:
Nice challenge! Do we have a solution though?

I don't have a lot of experience with this. But my solution would be to have issues like 'Open up an issue on Open Source Project X for problem Y'  and two linked ones: 'Submit pull request on Open Source project B for project X' 'Implement comments on PR Z' In other words make sure that you can commit to the acceptance criteria of the work-item, which in open source projects means splitting investigation, implementation and review process.

Reply:
Or maybe making sure that maintainers of the open-source project are aware of the sprint that you are running.

Reply:
Yes, I think you're probably right. The answer is that many/most open source projects clearly do follow a sort of loose Agile process in the use of issues, PRs etc. You need to identify what parts of the project you have "ownership" of, and which parts you don't, and then plan development to mitigate issues. A bit like the PerfectFit project, I suppose? Although goaliejs is obviously not open source.

Reply:
Yeah, only the waiting time is predictably unpredictable. If you want, please add this solution to the discussion on solution.

Reply:
OK I've finally done this. Better late than never...